functors task (1)/student/src/impl.cpp

```diff
@@ -1,2 +1,22 @@
 #include "impl.h"
 
+constexpr int count = 6;
```

Comment:
std::array has .size() method

Reply:
this method do not solve anything, because we still need to declare array with a constant:
std::array<BeerBrand, count>

std::make_array is still in proposal.
Honestly I wanted to use c-array with std::size, but this is available since c++17
Maybe there is better to use std::vector with initialization list

Reply:
Yes, but here it is still a magic number.
You can get size from casting enum values.